docs/source/models/cause.rst

```diff
@@ -171,29 +174,321 @@ Common data sources for cause models
 
 .. todo::
 
-   Format as table with measure, measure definition, data sources and
-   their uses.
+   Update mortality-related data sources within existing format.
+
+Once a cause model structure is specified, data is needed to inform its states
+and transitions. For our purposes, cause models generally have the following
+data needs:
+
+#. The probability that a simulant will start the simulation in a given state
+   within the cause model.
+#. The probability that a simulant will transition to a new state within the
+   cause model in a given time-step.
+#. The disability weight for each state in the cause model
+#. The probability that a simulant in a given cause model state will die in a
+   given time-step.
+
+There are several common data sources that can be used for these needs, which
+are outlined in the table below and discussed in more detail afterward.
+`Non-standard data sources`_ are discussed later on this page.
+
+.. list-table:: Data Definitions
+   :widths: 20 30 30
+   :header-rows: 1
+
+   * - Measure
+     - Definition
+     - Uses
+   * - `Prevalence`_
+     - Proportion of population with a given condition.
+     - Initialize cause model states.
```

Comment:
I think the Uses column of this table should describe what the quantity is used as rather than what it is used for.

E.g. Prevalence is the proportion of the population with a condition and it is used as the probability that an individual enters the simulation with the condition.

Reply:
@collijk What do you mean by "used as" vs. "used for"? Do you mean we should be more specific about how we are using prevalence to initialize model states?

Reply:
I mean we use, e.g., the prevalence *as* the probability that an individual has a condition when they enter which is used *for* initialization of the cause model states.

Reply:
I updated this in my most recent commit